README.md

```diff
@@ -84,9 +84,14 @@ a new branch to the `looker-hub` repository:
 ```
 export HUB_BRANCH_PUBLISH="yourname-generation-test-1"
 export GIT_SSH_KEY_BASE64=$(cat ~/.ssh/id_rsa | base64)
+export UPDATE_SPOKE_BRANCHES=false
+
 make build && make run
 ```
 
+To prevent PRs from automatically getting opened against spoke-default and spoke-private,
+while testing local changes, `UPDATE_SPOKE_BRANCHES` can be set to `false`.
```

Comment:
I think this should be the default behaviour, which would remove the need for us to document it here as part of the developer workflow (and risk people making mistakes). This a little more effort (since it will require modifying our deployment logic) but I think it will be worth it in the long term.

Reply:
Yeah, I think that makes sense.

Reply:
I updated the Airflow task and set `UPDATE_SPOKE_BRANCHES` to false by default.